platforms/quarkus/api/src/main/resources/application.properties

```diff
@@ -10,6 +10,7 @@
 %dev.quarkus.oidc.auth-server-url=${APICURIO_KC_AUTH_URL:http://localhost:8090/auth/realms/apicurio}
 %dev.quarkus.oidc.client-id=${APICURIO_KC_CLIENT_ID:apicurio-api}
 
+%dev.quarkus.http.port=${HTTP_PORT:8080}
```

Comment:
I don't see much value in adding this since this is already possible by using `QUARKUS_HTTP_PORT` or -Dquarkus.http.port.